plugin.audio.intergalacticfm/addon.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.intergalacticfm" name="Intergalactic FM" version="3.0.0" provider-name="Dreamer, Pander">
+<requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="2.19.1"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+    <provides>audio</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">No Station Such Dedication</summary>
```

Comment:
en_GB

Reply:
Why `en_GB` and not `en_US`?

Reply:
done

Reply:
`en_GB` is kodi default. You haven't changed this...at least it doesn't show changed on the current revision.